### ISSUE TITLE
`ActiveParticipantsEvent` is not parsed correctly

### ISSUE THREAD

**Issue by vodemn** [voxeet/voxeet-sdk-android#20]
I've noticed, that even though `WebSocketProxy` and `VoxeetDispatcher` show participants' statuses as `ON_AIR`:
```
D/WebSocketProxy(22720): onMessage: websocket:=okhttp3.internal.ws.RealWebSocket@446129a message:={"type":"ActiveParticipantsEvent","thirdPartyId":"JXwaZEUhMdGNiHIUZMzAyQ==","conference":{"confId":"...","confAlias":"..."},"participantCount":2,"participants":[{"userId":"...","externalId":"...","externalName":"Dr. Turko","participantStatus":"ON_AIR","participantType":"USER"},{"userId":"...","externalId":"...","externalName":"Saruman White","participantStatus":"ON_AIR","participantType":"USER"}],"viewerCount":0}
D/VoxeetDispatcher(22720): dispatch: SOCKET_TEXT := {"type":"ActiveParticipantsEvent","thirdPartyId":"JXwaZEUhMdGNiHIUZMzAyQ==","conference":{"confId":"...","confAlias":"..."},"participantCount":2,"participants":[{"userId":"...","externalId":"...","externalName":"Dr. Turko","participantStatus":"ON_AIR","participantType":"USER"},{"userId":"...","externalId":"...","externalName":"Saruman White","participantStatus":"ON_AIR","participantType":"USER"}],"viewerCount":0}
D/EventsFactory(22720): decode: event := ActiveParticipantsEvent
```
but in the received event those are actually `ERROR`:
```kotlin
Log.d("ActiveParticipantsEvent", "Decoded ${event.participants}")
```

```
D/ActiveParticipantsEvent(22720): Decoded [Participant{id='...', participantInfo=ParticipantInfo{name='Dr. Turko', externalId='...', avatarUrl=''}, status=ERROR}, Participant{id='...', participantInfo=ParticipantInfo{name='Saruman White', externalId='...', avatarUrl=''}, status=ERROR}]
```

**Comment by vodemn** [comment]
> What version of the SDK are you on?

`io.dolby:sdk:3.8.2`

**Comment by FabienLavocat** [comment]
Thank you for the confirmation @vodemn and apologies for this late answer. I am able to reproduce the problem where the status is not properly converted, I am filing an internal ticket and we will issue a patch asap.

**Comment by FabienLavocat** [comment]
Note that the following code returns the proper status for the participants when `ActiveParticipantsEvent` is triggered:
```java
List<Participant> ps = VoxeetSDK.conference().getParticipants();
```

**Comment by codlab** [comment]
We released the 3.8.4 version which focuses onto the event issue. It's should working as expected now. edit : closing it for now, in case of issue with the event please feel free to reopening it
